gdal/configure

```diff
@@ -34481,6 +34481,273 @@ fi
 
 
 
+# Check whether --with-grok was given.
```

Comment:
The PR should not include chnages to configure.  It is sufficient to include the updates to configure.ac.

Reply:
For continuous integration purposes, as configure in still in git and we do not run autoconf, we still commit the regenerated configure file when we touch configure.ac

Reply:
Thanks, I have left the configure changes in the PR